### PR TITLE
Updated transfer_tokens.sh script

### PR DESCRIPTION
**Pull request by shahnawaz-creator**

### CODE REVIEW

File: 0chain/transfer_tokens.sh [hunk 1]
```diff
@@ -2,43 +2,35 @@
...
-        echo "zwallet binary already present"
+if [[ -f bin/zwallet ]] ; then
+    echo "zwallet binary already present"
+else
+    ubuntu_version=$(lsb_release -rs | cut -f1 -d'.')
+    if [[ ${ubuntu_version} -eq 18 ]]; then
+        echo "Ubuntu 18 is not supported"
+        exit 1
+    elif [[ ${ubuntu_version} -eq 20 || ${ubuntu_version} -eq 22 ]]; then
+        curl -L "https://github.com/0chain/zcnwebappscripts/raw/as-deploy/0chain/artifacts/zwallet-binary.zip" -o /tmp/zwallet-binary.zip
```
Comment (boddumanohar) [inline 0chain/transfer_tokens.sh:25]: please make sure that this binary is built with latest v1.11.1 tags

**Review by boddumanohar — APPROVED** [review]

**Review by dabasov — APPROVED** [review]
lgtm

**Review by peterlimg — APPROVED** [review]
lgtm

File: 0chain/transfer_tokens.sh (hunk 1 above)
Comment (shahnawaz-creator) [reply 0chain/transfer_tokens.sh:25]: Nope, this build id of older version. For I will need to test the build on both ubuntu 20, ubuntu22 both.

Comment (shahnawaz-creator) [reply 0chain/transfer_tokens.sh:25]: i tested the binary token transfer and staking is working fine

**Pull request merged**